### DISCUSSION TITLE
conflicting dependencies while building docker image of celery

### DISCUSSION THREAD

**Discussion by Sauraj1234** [celery/celery#7580] [General]
While setup of celery in docker i am getting dependencies issue billiard, kombu ,vine

Input : 

Run command : **docker-compose build celery**

Output : 

<img width="1762" alt="Screenshot 2022-06-17 at 11 44 22 AM" src="https://user-images.githubusercontent.com/31471612/174236844-0d08995f-286e-4592-aabf-391ca2ab8ee1.png">

Reference followed : 

<img width="1214" alt="Screenshot 2022-06-17 at 11 41 21 AM" src="https://user-images.githubusercontent.com/31471612/174236967-8cf8623a-ca3b-44e9-af3e-aee99ee2b319.png">

**Comment by auvipy** [comment]
pull from master branch of celery, this problem should have been fixed now

**Reply by Sauraj1234** [reply]
@auvipy Sorry but still getting same error .. 
thing which i have done get the pull from master 
<img width="1792" alt="Screenshot 2022-06-17 at 1 17 07 PM" src="https://user-images.githubusercontent.com/31471612/174252015-fd52893b-126f-4280-bfe0-91aec82dd7fd.png">


**git pull --rebase upstream master** >> Already up to date 
also did 
**git pull origin master**

Sorry if i am doing something wrong,Please consider that its my first contribution

**Reply by auvipy** [reply]
then try git pull  upstream master

**Reply by Sauraj1234** [reply]
@auvipy Still same issue
